core/baseline-core-v1.0-psd01.md

```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
 
-Note that a workgroup participant may be a user of another BPI than the BPI the workgroup is defined in. The appropriate authentication and authorization policies to enable such a scenario are up to implementers and beyond the scope of this document.
+**[R77]**	A BPI User MUST be capable of performing cryptographic secret sharing based on BPI specified cryptographic algorithms.
 
-**[RXX]** A workgroup MUST have at least one administrator.
+An example of cryptographic secret sharing is called ["Shamir Secret Sharing"](####Shamir).
 
-**[RXX]** A workgroup MUST have at least one security policy.
+**[R78]** An external BPI User MUST at least be able to create, read, update and delete the following BPI core components following business rules for each component that were established by the BPI operator and agreed to by BPI Users:
+* A BPI Account belonging to the BPI User. Note that a BPI Account as defined in Section 6 is different from a BPI User account as defined in [Section 5.2](#52-BPI-User-Account).
+* A BPI Workgroup as defined in [Section 6](#-6-Agreement-Execution). 
+* A BPI Workflow as defined in [Section 6](#-6-Agreement-Execution).
+* A BPI Workstep as defined in [Section 6](#-6-Agreement-Execution).
+* A BPI Transaction as defined in [Section 6](#-6-Agreement-Execution). 
 
-Note that a security policy consists of authentication and authorization rules for the workgroup participants. Note also that one or more workgroup administrators define the workgroup security policy.
+There may be other BPI components such as role definitions or security policies. The enablement of additional components in a BPI beyond this standard is left to each specific implementation.
 
-**[RXX]** A workgroup MUST have at least one privacy policy.
+**[R79]** An internal BPI User MUST be able to create, read, update and delete all components of a BPI.
 
-A privacy policy the data visibility rules for each participant.
+**[R80]** A BPI User MUST be able to send and receive BPI messages from other BPI users using the BPI Communication capability as defined in [Section 5.3](#53-BPI-Communication).
 
-**[RXX]** A workgroup administrator MUST be able to perform at minimum the following functions:
-* add or remove one or more participants
-* create, update and delete both security and privacy policies.
-* delete or archive a workgroup
+**[R81]** A BPI User MUST be able to create, read BPI messages from other BPI users using the BPI Communication capability as defined in [Section 5.3](#53-BPI-Communication).
 
-**[OXX2]** A workgroup MAY have more than one administrator.
+Note that when a BPI User executes these capabilities they are understood to be enabled by services under the direct custody of the BPI User. However, the BPI User may delegate these capabilities to the BPI the BPI user operates in, called a custodial approach.  
```

Comment:
It would be easier to read to improve `to the BPI the BPI user operates`. Thanks :)

Reply:
`to the BPI, in which the BPI user operates in`